src/RepositoryApiClient.cs

```diff
@@ -103,8 +103,9 @@ public static IRepositoryApiClient CreateFromAccessKey(string servicePrincipalKe
         /// <returns></returns>
         public static IRepositoryApiClient CreateFromLfdsUsernamePassword(string username, string password, string organization, string repoID, string baseUrl)
         {
-            var httpRequestHandler = new LfdsUsernamePasswordHandler(username, password, organization, repoID, baseUrl);
-            return CreateFromHttpRequestHandler(httpRequestHandler, baseUrl);
+            string baseUrlEndWithSlash = baseUrl.TrimEnd('/') + "/";

```

Comment:
could move this line to `CreateFromHttpRequestHandler` for the `baseUrlDebug` property since it seems like this fixes url problems if `baseUrlDebug` is passed into other constructors like `CreateFromAccessKey`